app/src/main/java/com/epam/caloriecalc/data/remote/RetrofitHelper.kt

```diff
@@ -0,0 +1,15 @@
+package com.epam.caloriecalc.data.remote
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitHelper {
```

Comment:
I don't think this file is necessary. It's unused and *Helper files are usually signalizing some kind of code smell, here what I can see is that the only thing this is doing is the instantiation of the Retrofit, which would have a perfect place in a provides function in a Dagger or Hilt module.